charts/stable/ring-mqtt/values.yaml

```diff
@@ -1,43 +1,50 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: tccr.io/truecharts/ring-mqtt
+  tag: 5.0.3@sha256:a089c8c1514c5c80cc2fc1fa529f633290538a963df5e3d0c7c93b14fd92b633
+
+podSecurityContext:
+  runAsGroup: 0
+  runAsUser: 0
+
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: false
+
+secretEnv:
+  RINGTOKEN: ""
+  MQTTUSER: ""
+  MQTTPASSWORD: ""
+
 env:
   BEAMDURATION: "0"
   DISARMCODE: ""
   ENABLECAMERAS: "false"
   ENABLEMODES: "false"
   ENABLEPANIC: "false"
   MQTTHOST: ""
-  MQTTPASSWORD: ""
   MQTTPORT: "1883"
-  MQTTUSER: ""
   RINGLOCATIONIDS: ""
-  RINGTOKEN: ""
   SNAPSHOTMODE: disabled
-image:
-  pullPolicy: IfNotPresent
-  repository: tccr.io/truecharts/ring-mqtt
-  tag: latest@sha256:fe9beb0313fd81f26e698e7b528ae13eab5e74f65fade8d51f626c3fc272e066
-persistence:
-  data:
-    enabled: true
-    mountPath: /data
-podSecurityContext:
-  runAsGroup: 0
-  runAsUser: 0
-probes:
-  liveness:
-    enabled: false
-  readiness:
-    enabled: false
-  startup:
-    enabled: false
-securityContext:
-  readOnlyRootFilesystem: false
-  runAsNonRoot: false
+
 service:
   main:
-    enabled: false
     ports:
       main:
-        enabled: false
+        protocol: HTTP
+        targetPort: 55123
+        port: 55123
+  rtsp:
+    ports:
+      rtsp:
+        protocol: TCP
+        targetPort: 8554
+        port: 8554
+
+persistence:
+  data:
+    enabled: true
+    mountPath: /data
 
 portal:
   enabled: false
```

Comment:
```suggestion
  enabled: true
```

Reply:
Its temp for authorization, once it's done,  no longer works

Reply:
That's why i left it without a button